Index: sys/arm/ti/omap4/omap4_prcm_clks.c
===================================================================
--- sys/arm/ti/omap4/omap4_prcm_clks.c  (revision 275611)
+++ sys/arm/ti/omap4/omap4_prcm_clks.c  (working copy)
@@ -1336,8 +1336,9 @@
        return (0);
 }

-#define PRM_RSTCTRL            0x1b00
-#define PRM_RSTCTRL_RESET      0x2
+#define PRM_RSTCTRL                    0x1b00
+#define PRM_RSTCTRL_RST_GLOBAL_WARM    0x1
+#define PRM_RSTCTRL_RST_GLOBAL_COLD    0x2

 static void
 omap4_prcm_reset(void)
@@ -1344,7 +1345,7 @@
 {
        struct omap4_prcm_softc *sc = omap4_prcm_sc;
        bus_write_4(sc->sc_res[0], PRM_RSTCTRL,
-           bus_read_4(sc->sc_res[0], PRM_RSTCTRL) | PRM_RSTCTRL_RESET);
+           bus_read_4(sc->sc_res[0], PRM_RSTCTRL) | PRM_RSTCTRL_RST_GLOBAL_WARM);
        bus_read_4(sc->sc_res[0], PRM_RSTCTRL);
 }

