Index: sys/arm/ti/ti_i2c.c
===================================================================
--- sys/arm/ti/ti_i2c.c	(revision 274385)
+++ sys/arm/ti/ti_i2c.c	(working copy)
@@ -116,7 +116,7 @@
 	{ IIC_UNKNOWN,	 100000, 23, 13, 15,  0, 0},
 	{ IIC_SLOW,	 100000, 23, 13, 15,  0, 0},
 	{ IIC_FAST,	 400000,  9,  5,  7,  0, 0},
-	{ IIC_FASTEST,	1000000,  5,  3,  4,  0, 0},
+	{ IIC_FASTEST,	 400000,  9,  5,  7,  0, 0},
 	/* { IIC_FASTEST, 3200000,  1, 113, 115, 7, 10}, - HS mode */
 	{ -1, 0 }
 };
